recirq/quantum_chess/circuit_transformer.py

```diff
@@ -341,7 +346,12 @@ def qubit_mapping(self, circuit: cirq.Circuit) -> Dict[cirq.Qid, cirq.GridQubit]
         self.mapping = mapping
         return mapping
 
-    def transform(self, circuit: cirq.Circuit) -> cirq.Circuit:
+    def __call__(
```

Comment:
Decorate the class with `@cirq.transformer` to add support for automated logging